test-app/src/widgets/Draggable.js

```diff
@@ -73,8 +74,11 @@ export default class Draggable extends createjs.Container {
   moveToTarget(evt) {
     const { currentTarget } = evt;
     const dropTarget = _.find(this.dropTargets, drop => drop.label === currentTarget.label);
+
     if (!dropTarget.placed) {
       this.drop(dropTarget);
+    } else if (this.targetId === dropTarget.id) {
```

Comment:
Because of changes to `swap` and `revert` to support moving draggables around in the DOM, this case is now necessary instead of just letting swap account for it